src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,12 @@
 package mate.academy.lib;
 
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.FIELD)
```

Comment:
```suggestion
@Target(ElementType.FIELD)
```
you can remove this field